admin/common_auth/models.py

```diff
@@ -73,7 +73,14 @@ class Meta:
     #     return self.is_admin
 
     @property
+    def full_name(self):
```

Comment:
I'm seeing:
```
In [5]: u.get_full_name()
Out[5]: u'foo@bar.com'

In [6]: u.get_short_name()
Out[6]: u'foo@bar.com'
```
in my django shell. 

Do you think it's a good idea to override these methods?